src/Models/AbstractEntity.php

```diff
@@ -189,7 +189,7 @@ public function createFromTemplate(int $templateId, ?string $title = null): int
         $this->ItemsLinks->duplicate($templateId, $id, true);
         $this->ExperimentsLinks->duplicate($templateId, $id, true);
         $CompoundsLinks = LinksFactory::getCompoundsLinks($this);
-        $CompoundsLinks->duplicate($templateId, $id, fromItemsTypes: true);
+        $CompoundsLinks->duplicate($templateId, $id, true);
```

Comment:
use named arguments here: when reading the code, it's easier to see `fromTemplate: true` instead of just `true` -> we immediatly know what this flag is about